drivers/src/make_overlay_dts_orin-devkit.py

```diff
@@ -2937,10 +2937,10 @@ def get_serdes_pix_clk(str_revision, str_camera_type):
 
 if l4t_revision == "R35.1":
     str_rev_num = "351"
-elif l4t_revision == "R35.2.1":
+elif l4t_revision == "R35.2.1" or "R35.4.1":
```

Comment:
The expression here always evaluates to `True`, which leads to a bug.
```
elif l4t_revision == "R35.2.1" or l4t_revision == "R35.4.1":
```

Reply:
Thanks. Fixed.